### DISCUSSION TITLE
Add a Profile Badge (like Modrinth staff) for lead devs for Quilt, Fabric etc

### DISCUSSION THREAD

**Discussion by Dontknow09** [modrinth/.github#155] [Feature requests]
I think this would both make it much easier to find users of trusted organizations and also just a fun thing to see on someones profile if you decide to add it.

**Comment by triphora** [comment]
Likely would be redundant once we add creator orgs

**Reply by Dontknow09** [reply]
Oh, sorry didn't know you were adding that. So if I may ask how is it gonna work then?
Like how are you gonna check if a certain organization (think thats what you mean with orgs) is something that you actually want to show off on your platform?

**Reply by triphora** [reply]
Not sure what you mean by "show off", but the organizations a member is part of would be displayed on their profile like GitHub

**Reply by Dontknow09** [reply]
I mean how do you filter out the people who are gonna put explicit content in their organizations? And is it automatic like it takes the info from GitHub or how do I add my own organization?

**Reply by triphora** [reply]
These are implementation details that have not yet been entirely worked out. Moderation will be the same as with users, though.

**Comment by triphora** [comment]
Organizations have been added. Closing as such.
